### DISCUSSION TITLE
timed-out reading channel data not available

### DISCUSSION THREAD

**Discussion by mohit-singhal20** [ktbyers/netmiko#3059] [General]
Hello ,

            switch = {
                            'ip':           routerIp,
                            'device_type':  deviceType,
                            'port':         routerPortFirewall,
                            'username':     userName,
                            'password':     password,
                            'secret':       password,
                            'global_delay_factor': globalDelayFactor
                    }
            
            blockIpList = open(blockIpsFilePath,"r")
            blockIpList = blockIpList.read().split("\n")
            ##### Connecting Router #####
            switchConnect = ConnectHandler(**switch)
            switchConnect.enable()
            switchConnect.config_mode()
            configCommands = ["object-group network ipv4"]
            switchConnect.send_config_set(configCommands,exit_config_mode=False)
            for validIP in blockIpList:
                if(validIP != ''):
                    validIP = validIP.strip('\n')
                    cmd = 'host ' + validIP
                    resultVerify = switchConnect.send_command_expect(cmd, expect_string=r'#', delay_factor=10 )
            switchConnect.send_command_expect("commit", delay_factor=10)
            switchConnect.exit_config_mode()
            switchConnect.disconnect()

When executing script for same device sometime I am getting '**timed-out reading channel data not available**' and sometime it is working fine , I am using port 22

**Comment by nimishb98** [comment]
I am getting the same error. Any help?
